sources/models/contacts.js

```diff
@@ -1,4 +1,12 @@
-export const contacts = [
-	{"id":1,"Name":"Alex Wanny","Email":"alex@gmail.com","Status":1,"Country":2},
-	{"id":2,"Name":"Doris Wan","Email":"doris@gmail.com","Status":1,"Country":3}
-];
\ No newline at end of file
+// export const contacts = [
+// 	{"id":1,"Name":"Alex Wanny","Email":"alex@gmail.com","Status":1,"Country":2},
+// 	{"id":2,"Name":"Doris Wan","Email":"doris@gmail.com","Status":1,"Country":3}
+// ];
```

Comment:
remove commented lines

Reply:
I know it, sorry. Tested a lot of different features and in the end forgot to remove them.